tests/helm/templates/nouveau.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: nouveau
+  name: cht-couchdb-nouveau
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: nouveau
+  template:
+    metadata:
+      labels:
+        cht.service: nouveau
+    spec:
+      containers:
+      - name: cht-couchdb-nouveau
+        image: {{ $.Values.upstream_servers.docker_registry }}/cht-couchdb-nouveau:{{ $.Values.cht_image_tag }}
+        ports:
+        - containerPort: 5984
+        volumeMounts:
+        - mountPath: /data/nouveau
+          name: couchdb-nouveau-claim0
+          subPath: data
+      restartPolicy: Always
+      volumes:
+      - name: couchdb-nouveau-claim0
+        persistentVolumeClaim:
+          claimName: couchdb-nouveau-claim0
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    cht.service: nouveau
+  name: nouveau
+spec:
+  ports:
+    - name: nouveau-service
+      port: 5987
```

Comment:
The port here is conflicting with the `containerPort` set in the `Deployment`, is this correct?